--- origsrc/libmodman-2.0.1/libmodman/CMakeLists.txt	2010-09-01 15:19:38.000000000 -0500
+++ src/libmodman-2.0.1/libmodman/CMakeLists.txt	2010-11-10 13:30:29.354562200 -0600
@@ -49,8 +50,10 @@ add_library(modman SHARED
 if(NOT WIN32)
   target_link_libraries(modman dl)
 endif()
-set_target_properties(modman PROPERTIES PREFIX "lib" VERSION 1.0.0 SOVERSION 1)
-install(TARGETS modman DESTINATION ${LIB_INSTALL_DIR})
+set_target_properties(modman PROPERTIES VERSION 1.0.0 SOVERSION 1)
+install(TARGETS modman LIBRARY DESTINATION ${LIB_INSTALL_DIR}
+                       RUNTIME DESTINATION ${BIN_INSTALL_DIR}
+                       ARCHIVE DESTINATION ${LIB_INSTALL_DIR})
 install(FILES   module_manager.hpp module.hpp DESTINATION ${INCLUDE_INSTALL_DIR}/${PROJECT_NAME})
 
 ### Tests
